<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Apply a background color to the central widget */
QWidget#centralwidget {
    background-color: #FFE4B5;
}

/* Style the QLineEdit for username in the Login section */
QLineEdit#usernameLoginLineEdit,
QLineEdit#passwordLoginLineEdit,
QLineEdit#usernameRegisterLineEdit,
QLineEdit#passwordRegisterLineEdit,
QLineEdit#checkPasswordLineEdit {
    background-color: #C0C0C0;
    border: 1px solid #808080;
    width: 40%;
    height: 5%;
    transition: background-color 0.3s, border-color 0.3s;
}

/* LineEdit hover effect */
QLineEdit#usernameLoginLineEdit:hover,
QLineEdit#passwordLoginLineEdit:hover,
QLineEdit#usernameRegisterLineEdit:hover,
QLineEdit#passwordRegisterLineEdit:hover,
QLineEdit#checkPasswordLineEdit:hover {
    background-color: #D3D3D3;
    border-color: #555555;
}

/* LineEdit active effect */
QLineEdit#usernameLoginLineEdit:focus,
QLineEdit#passwordLoginLineEdit:focus,
QLineEdit#usernameRegisterLineEdit:focus,
QLineEdit#passwordRegisterLineEdit:focus,
QLineEdit#checkPasswordLineEdit:focus {
    background-color: #FFFFFF;
    border-color: #000000;
}

/* Style the QPushButton for Sign In in the Login section */
QPushButton#signInPushButton,
QPushButton#signUpPushButton {
    background-color: #1E90FF;
    color: #ffffff;
    border: none;
    padding: 1% 2%;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 2%;
    cursor: pointer;
    border-radius: 4px;
    transition: background-color 0.3s, border-color 0.3s, color 0.3s;
}

/* Button hover effect */
QPushButton#signInPushButton:hover,
QPushButton#signUpPushButton:hover {
    background-color: #007bff;
}

/* Button active effect */
QPushButton#signInPushButton:pressed,
QPushButton#signUpPushButton:pressed {
    background-color: #0056b3;
}

/* Style the QLabel for additional information in the Login section */
QLabel#label,
QLabel#label_2 {
    font-weight: bold;
    color: #333333;
}

/* Style the QCheckBox for Show Password in the Register section */
QCheckBox#showPasswordCheckBox {
    color: #333333;
}

/* Apply a background color to the central widget */
QWidget#centralwidget {
    background-color: #FFE4B5;
}

/* Style the QMenuBar */
QMenuBar#menubar {
    background-color: #B0C4DE;
    border: 1px solid #4682B4;
}

/* Style the QStatusBar */
QStatusBar#statusbar {
    background-color: #B0C4DE;
    border: 1px solid #4682B4;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="usernameLoginLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nazwa użytkownika</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Zaloguj się:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLoginLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Hasło</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signInPushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Zaloguj się</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Zarejestruj się:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameRegisterLineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nazwa użytkownika</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordRegisterLineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Hasło</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="checkPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Potwierdź hasło</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signUpPushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Zarejestruj się</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showPasswordCheckBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokaż hasło</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
